content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -117,7 +117,7 @@ Wait another 30 seconds, and verify that the container has been restarted:
 kubectl get pod liveness-exec
 ```
 
-The output shows that `RESTARTS` has been incremented:
+The output shows that `RESTARTS` has been incremented. Note that the `RESTARTS` counter increments as soon as a failed container in a restarts :
```

Comment:
The restarts counter only gets incremented if a pod fails for some reason and comes back to a running state again since it indicates the number of dead containers that have not yet been removed.

Thus, in my opinion this could be better worded as:

```suggestion
The output shows that `RESTARTS` has been incremented. Note that the `RESTARTS` counter increments as soon as a failed container comes back to the running state:
```